dmd-config/dmd.conf

```diff
@@ -1,5 +1,5 @@
 [Environment32]
-DFLAGS=-I%@P%/../import/druntime -I%@P%/../import/phobos -L-L%@P%/../lib32 -L--export-dynamic
+DFLAGS=-I%@P%/../import/druntime -I%@P%/../import/phobos -L-L%@P%/../lib32 -L--export-dynamic -L-rpath=/snap/dmd/current/lib32 -L--enable-new-dtags
```

Comment:
Can you explain the reason why the `--enable-new-dtags` flag is useful/needed here?

If it's for a reason other than supporting the `rpath`, it should probably be done in a separate patch.  Either way there should be a clear explanation in the commit message of why this is done.

Reply:
I think I have explained it in this commit https://github.com/dlang-snaps/dmd.snap/pull/31/commits/6b94eeb1b147a9d374c4cd40d414c49b45b7416c

Reply:
Ah, sorry, missed that.  Looks good.